index.html

```diff
@@ -5064,7 +5064,7 @@ <h3>Using the Document Base for the Default Vocabulary</h3>
 
   <p>The example above would associate <em>忍者</em> with the specified default
     language code <code>ja</code>, <em>Ninja</em> with the language code
-    <code>en</code>, and <em>Nindža</em> with the language code <code>cs</code>.
+    <code>en</code>, and <em lang="cs">Nindža</em> with the language code <code>cs</code>.
     The value of <code>name</code>, <em>Yagyū Muneyoshi</em> wouldn't be
     associated with any language code since <code>@language</code> was reset to
     <a>null</a> in the <a>expanded term definition</a>.</p>
```

Comment:
you added only the one `lang` attribute (for Czech). Did you mean to add others here?

Reply:
Fixed in 9b1e522554ac86b8079f21b10ca254a2c478ec2a.